types/react/test/experimental.tsx

```diff
@@ -26,6 +49,23 @@ function useExperimentalHooks() {
     // $ExpectType () => string
     const deferredConstructible = React.useDeferredValue(Constructible);
 
+    // $ExpectType string
+    const pathname = React.useMutableSource(
+        // $ExpectType MutableSource<Window, string>
```

Comment:
I don't think this is very important. It's just repeating how you called it before. `$ExpectType` break from my experience all the time without indicating an actual issue.

The only important part I can see is making sure that `pathname` is a string and `const pathname: string = React.useMutableSource(...)` should be sufficient.